web/source/kmwembedded.ts

```diff
@@ -424,9 +424,35 @@
       var Lelem=keymanweb.domManager.getLastActiveElement(),Lkc,keyShiftState=osk.getModifierState(layer);
       
       keymanweb.domManager.initActiveElement(Lelem);
+
+      var nextLayer: string;
+
+      // This should be set if we're within this method... but it's best to guard against nulls here, just in case.
+      if(osk.popupBaseKey && osk.popupBaseKey.key) {
+        // This is set with the base key of our current subkey elsewhere within the engine.
+        var baseKey = osk.popupBaseKey.key;
+        var found = false;
+
+        // Search for the specified base key so we can retrieve its useful properties.
```

Comment:
Confusing to use "base key" here in this comment -- 'base key' refers to the key in the layer. Can we use a different term?

Reply:
I mean, I suppose so... but should I then refactor the existing field `popupBaseKey` to match?